tasks-by-users/Rehan-Khan-Engineer/06-Loading-Animation-CSS/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #262626;
+  /* height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center; */
+}
+
+.jj-ring {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  height: 150px;
+  width: 150px;
+  border: 3px solid #3c3c3c;
+  background: transparent;
+  text-align: center;
+  text-transform: uppercase;
+  line-height: 150px;
+  border-radius: 50%;
+  color: yellow;
+}
+
+.jj-ring::before {
+  content: "";
+  position: absolute;
+  top: -1.5px;
+  left: -1.5px;
+  height: 150px;
+  width: 150px;
+  border-top: 3px solid yellow;
+  border-right: 3px solid yellow;
+  border-radius: 50%;
+  animation: long-fish 2s linear infinite;
+}
+
+span {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  height: 3px;
+  width: 50%;
+  background: yellow;
+  background: transparent;
+  transform-origin: left;
+  animation: fish 2s linear infinite;
+}
+
+span:before {
+  content: "";
+  position: absolute;
+  top: 50%;
+  left: 100%;
+  transform: translate(-50%, -50%);
+  height: 12px;
+  width: 12px;
+  border-radius: 50%;
+  background: yellow;
+  box-shadow: 0 0 20px yellow;
```

Comment:
What is the 4th value of `box-shadow` property which you define as `40px` ??
How it effects the shadow ?
@Rehan-Khan-Engineer

Reply:
here 20px is  Blur effect  value, if you add 4th value as 20px, it will mean it is 20px spread of shadow.